subsystem/low_level_api_test.go

```diff
@@ -56,7 +56,53 @@ var _ = Describe("Low level API behaviours", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(string(responseBytes)).To(MatchJSON(`{
 			"code": 400,
-			"message": "parsing newClusterParams body from \"\" failed, because json: unknown field \"base_dns_doman\""
+			"message": "validation failure list:\nparsing newClusterParams body from \"\" failed, because json: unknown field \"base_dns_doman\""
```

Comment:
The addition of `validation failure list:\n` is due to the wrapping of the `ServeError` function